MPSGameLift/Scripts/export_gamelift_server_package.py

```diff
@@ -10,10 +10,14 @@
 
 To use this script pass it into o3de.bat's export-project command: 
 <path-to-o3de-engine>\scripts\o3de.bat export-project -es <path-to-multiplayer-sample>\MPSGameLift\Scripts\export_gamelift_server_package.py -ll INFO
+A folder named GameLiftPackageWindows containing the server will be created. 
+Test the server locally and upload the server package to GameLift. 
+    'aws gamelift upload-build --server-sdk-version 5.0.0 --operating-system WINDOWS_2016 --build-root .\GameLiftPackageWindows\ --name MultiplayerSample --build-version v1.0 --region us-west-2'
```

Comment:
It should be possible to pass in the `region` here, since the script needs to support all regions.

Reply:
@AMZN-Gene don't think this was addressed, is it planned as a follow-up?

Reply:
Didn't realize that was in the log, never mind :D